### DISCUSSION TITLE
Erro en identacion

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/CursoIntroPython#287] [Q&A] [answered]
Buena tarde compañero tengo un problema al ingresar el código en nivel de oxigen me indica el siguiente error

![Sin título](https://user-images.githubusercontent.com/68626555/154335678-ff6ea0ee-de2b-4b50-9b80-8ee04e5c5e44.png)

**Comment by MarcosSilvaG** [comment, marked as answer]
hola! La indentación son las tabulaciones o espacios que dejas en las líneas de código, para que este no suceda cuída los espaciados que dejas.
En este caso bastará conque borres todos los espacios antes de cada palabra que inicia tu línea de código.

**Comment by LuceroLuciano** [comment]
Hola! 

Me parece que debes eliminar los espacios(idenatación) que tienes al principio:
`import numpy as np`
`import matplotlib.pyploy as plt`
`data = np.random.default_rng(12345)`
`oxy_nums = data.integers(low=0, high=10, size=10)`
`plt.bar(range(len(oxy_nums)), oxy_nums)`
`ptl.show()`

Espero te ayude. 
Saludos :)

**Reply by gersondlarosa** [reply]
Muchas gracias y quedo
